api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -801,6 +801,26 @@ public Order discontinueOrder(Order orderToDiscontinue, String reasonNonCoded, D
 	 */
 	@Authorized(PrivilegeConstants.GET_ORDERS)
 	public OrderGroup getOrderGroup(Integer orderGroupId) throws APIException;
+
+	/**
+	 * Fetches all order groups for the specified patient
+	 * 
+	 * @param patient the patient to match on
+	 * @return list of matching OrderGroups
```

Comment:
Can you add since annotations to the new interface methods? That way, clients will easily tell the version when they were introduced.